main.go

```diff
@@ -186,6 +186,40 @@ func populateArgs(c *cli.Context) (
 	return
 }
 
+func overrideWithLoggingFlags(mountConfig *config.MountConfig, flags *flagStorage) {
+	// if log file is not set in config file, override it with flag value.
+	if mountConfig.LogFile == "" {
+		mountConfig.LogFile = flags.LogFile
+	}
+	// if log format is not set in config file, override it with flag value.
+	if mountConfig.LogFormat == "" {
+		mountConfig.LogFormat = flags.LogFormat
+	}
+	// if debug_fuse, debug_gcsfuse or debug_mutex flag is set, override log
+	// severity to TRACE.
+	if flags.DebugFuse || flags.DebugGCS || flags.DebugMutex {
+		mountConfig.LogConfig.Severity = config.TRACE
+	}
+}
+
+func resolveFilePath(filePath string, configKey string) (resolvedPath string, err error) {
+	resolvedPath, err = getResolvedPath(filePath)
+	if filePath == resolvedPath || err != nil {
+		return
```

Comment:
looks like this is all duplicate of code in flags.go. can we common logic into some utility file.

Reply:
Refactored the code.